curriculum/challenges/english/25-front-end-development/review-dictionaries-and-sets/67f39babe1e2ec1fb6eea32a.md

```diff
@@ -7,13 +7,381 @@ dashedName: review-dictionaries-and-sets
 
 # --description--
 
-## First topic
+## Dictionaries
 
-Describe
+- **Dictionaries**: Dictionaries are built-in data structures that store collections of key-value pairs. This is the general syntax of a Python dictionary:
 
-## Second topic
+```python
+dictionary = {
+    'key1': 'value1',
+    'key2': 'value2'
```

Comment:
```suggestion
    key1: value1,
    key2: value2
```
We should add a brief reminder that keys should be immutable types while values are not restricted